src/Simulation/Simulators/QuantumProcessor/ClassicalControl.cs

```diff
@@ -83,7 +83,7 @@ private static void RunClause(ICallable op, OperationFunctor type, IQArray<Qubit
         }
 
 
-        public class QuantumProcessorApplyIfElse : Extensions.ApplyIfElseIntrinsic
+        public class QuantumProcessorApplyIfElse : Quantum.Simulation.QuantumProcessor.Extensions.ApplyIfElseIntrinsic
```

Comment:
Since this is in the `Microsoft.Quantum.Simulation.QuantumProcessor` namespace, is the long name actually needed?

Reply:
Looks like it doesn't make a difference. I'll remove the extra qualifiers.

Reply:
done